_posts/2017/2017-03-22-microxchg-2017-microservices-conference.md

```diff
@@ -0,0 +1,89 @@
+Although I learned not really something new totally new, there where quite a lot of more details.
+Especially Fred's tellings of histories of his life and big, big software projects were worth every second I could spend sitting in the first row.
+I also had the opportunity to ask questions and chat with participants.
+
+## Workshop Day
+
+I visited the workshop building microservcies with Scala and Akka by [Yaroslav Tkachenko](http://sap1ens.com).
+It was an awesome experience.
+He is a highly skilled professional with a very long background in Scala and prepared a workshop that could have easily spanned two more days.
+
+The material until the very last seconds of the workshop visited the conference as well [afterwards](http://sap1ens.com/blog/2017/02/26/microxchg-2017), where we had many interesting discussions about the tech communities in Canada.
+
+## Visited Talks
+
+I could visit a lot of interesting sessions with a wide variety of topics.
+All talks where recorded with 4 microphones, 3 cameras in each room.
+The movie crew also had 2 setup setups on sight with 8 monitors.
+They could make professional movie cuts of the talks directly after the talks.
+Just 30 minutes after a talk was finished it was uploaded directly into familiar movie services.
+You can find them on the [microXchg YouTube Channel](https://www.youtube.com/channel/UCGCbB8TPtYMQmJwYVogcPjg).
+And also on [InfoQ](https://www.infoq.com/microxchg), which is one of the best platform for tech talks on the internet as slides and speaker recordings are presented in two windows.
+As you can watch all recordings there I will only give a summary on one single talk
+
+## Talks on Thursday
+
+**Morning:**
+
+* Resilient functional software design (Uwe Friedrichsen)
+* Shrinking Microservices to functions (Adrian Cockcroft)
+* DDD & REST - Domain-Driven APIs for the web (Oliver Gierke)
+* Serverless - Power to the Black Box! (Michael Bruns)
+
+**Afternoon:**
+
+* AaaS – Anything as a Service. Anything left to do, then? (Dustin Huptas)
+* Architecture Pattern for Microservices in Kubernetes (Dr. Thomas Fricke)
+* Microservices: The Organizational and People Impact (Daniel Bryant)
+* IoT and MicroServices in the Home (Fred George)
+
+## Talks on Friday
+
+**Morning:**
+
+* Authorization and Authentication on Microservice Environments (Bernd Schönbach was ill, so his colleague Dr. Niklas Henrich took over)
+* Secure Microservices Adoption (Grygoriy Gonchar)
+* Beyond OAuth2: end to end microservice security (Will Tran)
+
+**Afternoon:**
+
+* How to properly blame things for causing latency: An intodurction to distributed tracing and Zipkin (Adrian Cole)
+* Micorservies, Conway's Law and the Innovator's Dilemma (Andrea Stubbe)
+* How to monitor Microservices with dtrace and sysdig (Jorge Salamero)
+
+## Insights into "Beyond OAuth2" by Will Tran
+
+
+
+
+## Summary
+
+As you can tell the speaker lineup was incredible and the talks absolutly stunning.
```

Comment:
This reads better as a bullet list. Try it like that:

* Incredible speaker lineup and absolutely stunning talks
* Amazing atmosphere and throughout every single moment it was filled with so much energy.
* Absolutely the best conference I've ever visited.
* Orga team was a blast and I was honoured to being part of the extended crew.

I'm really happy that I could help making this conference a great experience and to also interview some of the top speakers.
Thanks for such terrific days in Berlin, the delicious catering and the vitalising discussions.
Special thanks to Oliver Gierke who ...